bibtex-actions.el

```diff
@@ -73,25 +126,55 @@
                (lambda (string predicate action)
                  (if (eq action 'metadata)
                      '(metadata
-                       ;; TODO (affixation-function . bibtex-actions--affixation)
+                       (when bibtex-actions-rich-ui
```

Comment:
This is the other problematic line; maybe the same problem.

I want this affixation function to run by default, but to allow a user to turn it off if they prefer.

Reply:
oh, Umm you probably wanted to evaluate that second sub-expression instead of inserting it as a quoted list. You can do either:

```lisp
`(metadata
  ,(when bibtex-actions-rich-ui
     '(affixation-function . bibtex-actions--affixation)))
```

or

```lisp
(list 'metadata
      (when bibtex-actions-rich-ui
        '(affixation-function . bibtex-actions--affixation)))
```

If you're unsure about the difference please ask :-).

Reply:
Yeah, I'm totally lost with the '`, stuff!

Everything was working fine before I introduced that single "when" line. I don't understand why fixing it would require syntax tweaks across multiple lines.

Reply:
Not sure why, but all fixed now.

And I did submit a PR for bibtex-completion; will remove the link symbol variable here if that's merged.